src/shared/contractOptionsTitle.tsx

```diff
@@ -0,0 +1,5 @@
+export enum ContractOptionsTitle {
```

Comment:
let's create a `enum` folder, and it can be just `.ts` since it is not returning react element